tests/Data/ProgrammesDb/EntityRepository/CoreEntityRepository/FindProgrammesByAncestryAndTypesTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository::<public>
+ * @group repository
+ * @group programmesAggregation
+ */
+class FindProgrammesByAncestryAndTypesTest extends AbstractDatabaseTest
+{
+    public function testFindEpisodeGuideChildrenGetCorrectClips()
```

Comment:
At first glance, I don't quite understand the test name. What is this test doing?